FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -135,7 +253,16 @@ public static function configDataProvider()
                 'findologicDI' => false,
                 'isSearchPage' => false,
                 'isCategoryPage' => true,
-                'expected' => false
+                'expected' => true
```

Comment:
This case wasn't fixed, but rather the condition was changed. Make sure this still returns `false`. You may need some more setup so the `Environment` returns what you need.